README.rst

```diff
@@ -108,154 +108,111 @@ Latest News
-* `Text to Speech <nemo/collections/tts/README.md>`_
-* `Computer Vision <nemo/collections/vision/README.md>`_
+* `NeMo Developer Docs < https://docs.nvidia.com/nemo-framework/user-guide/latest/nemotoolkit/index.html>`_
+* `NeMo Framework User Guide < https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html>`_
 
 Requirements
 ------------
 
-1) Python 3.10 or above
-2) Pytorch 1.13.1 or above
-3) NVIDIA GPU, if you intend to do model training
-
-Developer Documentation
------------------------
+* Python 3.10 or above
+* Pytorch 1.13.1 or above
+* NVIDIA GPU (if you intend to do model training)
 
-.. |main| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=main
-  :alt: Documentation Status
-  :scale: 100%
-  :target: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/
-
-.. |stable| image:: https://readthedocs.com/projects/nvidia-nemo/badge/?version=stable
-  :alt: Documentation Status
-  :scale: 100%
-  :target:  https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/
-
-+---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| Version | Status      | Description                                                                                                                              |
-+=========+=============+==========================================================================================================================================+
-| Latest  | |main|      | `Documentation of the latest (i.e. main) branch. <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/>`_                  |
-+---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| Stable  | |stable|    | `Documentation of the stable (i.e. most recent release) branch. <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/>`_ |
-+---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
-
-
-Getting help with NeMo
+Install NeMo Framework
 ----------------------
-FAQ can be found on NeMo's `Discussions board <https://github.com/NVIDIA/NeMo/discussions>`_. You are welcome to ask questions or start discussions there.
 
+NeMo Framework offers multiple installation options. Choose the option that is best suited to your specific domain requirements.
 
-Installation
-------------
+* The Conda/Pip installation method is recommended for ASR and TTS domains or when using NVIDIA PyTorch container as the base. See `Conda <#conda>`_ and `Pip <#pip>`_ for instructions.
 
-The NeMo Framework can be installed in a variety of ways, depending on your needs. Depending on the domain, you may find one of the following installation methods more suitable.
```

Comment:
This is not true for Non LLM/Multimodal. Please do not change this section, it has been carefully updated in the past.
Or be very explicit : 
ASR, TTS, NMT can be pip installed / conda installed
LLM, MM and CV should be used in docker container.

Reply:
OK

Reply:
Revert the changes. Will fix this issue in a separate PR.